ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -72,6 +93,36 @@ def __init__(self, tracer=None, config=None, service=None):
         except Exception:
             log.warning("Failed to load CODEOWNERS", exc_info=True)
 
+    def _check_enabled_features(self):
+        # type: () -> Tuple[bool, bool]
+        if not self._app_key:
+            return False, False
+        url = "https://api.datadoghq.com/api/v2/libraries/tests/services/setting"
```

Comment:
this should depend on the configured site `DD_SITE`.

Also: we should be able to do this request via EVP proxy, but this can be done at a later PR

Reply:
Will the host be `api.<DD_SITE>`?

Reply:
yeah